src/server.js

```diff
@@ -36,14 +36,15 @@ app.prepare().then(() => {
     [
       /(\/[a-zA-Z_-]+)?\/map\/.+/,
       /(\/[a-zA-Z_-]+)?\/community_support\/new.*/,
-      /(\/[a-zA-Z_-]+)?\/embed.*/,
       /(\/[a-zA-Z_-]+)?\/api\/docs.*/,
     ],
     (req, res) => {
       res.redirect(`http://classic.wheelmap.org${req.originalUrl}`);
     }
   );
 
+  server.get([/(\/[a-zA-Z_-]+)?\/embed.*/], (req, res) => res.redirect('/?embedded=true'));
```

Comment:
-2: make sure to keep old url params - lat, lon etc are stripped this way.

Reply:
Fixed